pkg/operator/ceph/cluster/controller.go

```diff
@@ -305,11 +305,28 @@ func (c *ClusterController) configureExternalCephCluster(namespace, name string,
 	specCephVersion := *specCephVersionImage
 
 	// Populate clusterInfo with the external cluster version
-	cluster.Info.CephVersion = *cephMonVersion
+	// This is tricky since this could lead to the impression that we upgrade, but we have not
+	// So we should probably set this somewhere else
+	//
+	// Note that the Ceph version we use as a reference is **always** the one from the external cluster
+	//
+	// Also we need to know if this is a brand new cluster or if other CRD are running too
+	// for the initial start, if the local version is < to external version, we fail right away
+	// However, if this is not a new cluster, meaning an object cr is running on the local cluster
+	// we can assume that the external cluster got upgraded, then it becomes acceptable if local version is lower from a minor version
+	//
+	cluster.Info.CephVersion = *externalCephMonVersion
+
+	// Do we have an existing CRs running already? (MDS, RGW, NFS)
+	r, err := otherCRResources(c.context, namespace)
```

Comment:
Could we just run this inside `validateCephVersionsBetweenLocalAndExternalClusters()` instead of passing as a param?

Reply:
If I do this, it becomes difficult to test.